requirements-build.txt

```diff
@@ -0,0 +1,2 @@
+cryptography==2.8
```

Comment:
Let's add a comment on why we need this file.

Reply:
Agreed. It's not immediately obvious. In general I'd like to avoid what we have with the bundled installer where when we update our dependencies you have to magically know to update the make-bundle script file as well. It would be nice if we had some better way than a comment in the build requirements to signal where you need to update dependencies in strange places.